index.html

```diff
@@ -1177,6 +1179,13 @@
             sync (unless <a>media mirroring</a> is used).
           </div>
           <div class="note">
+            The user agent is encouraged to pass locale information to the
+            <a>remote playback device</a> when possible, so that the <a>remote
+            playback device</a> may adjust its user interface and operational
+            functions to locale-specific attributes that reflect the user's
+            preferences.
+          </div>
+          <div class="note">
             The user agent SHOULD pause local audio and video output of the
```

Comment:
"The user agent is expected to pause"

Reply:
Wait, no, that's addressed in #102 (and actually, that's the note that gets dropped in #102, to make this text fully normative)

Reply:
OK, then we're all good.